server/routes.js

```diff
@@ -112,12 +112,29 @@ module.exports = [{
     }
   }
 }, {
-  // static routes using dist/, yay grunt
   method: 'GET',
-  path: '/assets/{param*}',
+  path: '/images/{param*}',
   handler: {
     directory: {
-      path: path.join(__dirname, '..', config.get('server.staticPath')),
+      path: path.join(__dirname, '..', config.get('server.staticPath'), 'images'),
+      listing: config.get('server.staticDirListing')
+    }
+  }
+}, {
+  method: 'GET',
+  path: '/scripts/{param*}',
+  handler: {
+    directory: {
+      path: path.join(__dirname, '..', config.get('server.staticPath'), 'scripts'),
+      listing: config.get('server.staticDirListing')
+    }
+  }
+}, {
+  method: 'GET',
+  path: '/styles/{param*}',
+  handler: {
+    directory: {
+      path: path.join(__dirname, '..', config.get('server.staticPath'), 'styles'),
```

Comment:
@nchapman sure, I'm fine with doing a generic wildcard under `/public`, or maybe you meant putting stuff like `/scripts` at the topmost level? Either way, the API is namespaced under `/v1`, so I don't think the catch-all router should be too complex. @pdehaan whatever works is fine; we'll wind up moving all the assets to some other CDN'd domain eventually anyhow :-)